controllers/storagecluster/cephcluster.go

```diff
@@ -847,10 +852,6 @@ func newStorageClassDeviceSets(sc *ocsv1.StorageCluster) []rookCephv1.StorageCla
 			annotations := map[string]string{
 				"crushDeviceClass": crushDeviceClass,
```

Comment:
In a previously-created cluster that set `replicated`, will it continue using `replicated` here? Or why not set the `DefaultDeviceClass` here, unless overridden by the user?

Reply:
Yes as once set the crushDeviceClass doesn't change so it won't have a problem. We shouldn't use the defaultDeviceClass as it kinda depends on the cephCluster CR status to determine that, and this crushDeviceClass annotation is decided before that, also if it's set it can't be changed further.